src/v2/Components/NavBar/Menus/DropDownMenu/DropDownMenu.tsx

```diff
@@ -49,37 +57,52 @@ export const DropDownNavMenu: React.FC<DropDownNavMenuProps> = ({
         ]}
       >
         <Flex width={breakpoints.xl} px={3}>
-          <Links py={3} mr={[3, 3, 5, 5]}>
+          <Links>
             <Box
+              flexGrow={1}
               display="flex"
               flexDirection="column"
-              height="100%"
               mr={[1, 1, 2, 2]}
               width={[110, 135, 135, 170, 170]}
             >
               {menu.links.map((menuItem, i) => {
                 if (!("menu" in menuItem)) {
-                  const isLast = menu.links.length - 1 === i
-
-                  return isLast ? (
-                    <ViewAllMenuItem key={menuItem.text} href={menuItem.href}>
-                      {menuItem.text}
-                    </ViewAllMenuItem>
-                  ) : (
-                    <LinkMenuItem key={menuItem.text} href={menuItem.href}>
-                      {menuItem.text}
-                    </LinkMenuItem>
+                  const isLast = lastMenuLinkIndex === i
+
+                  return (
+                    !isLast && (
+                      <LinkMenuItem key={menuItem.text} href={menuItem.href}>
+                        {menuItem.text}
+                      </LinkMenuItem>
+                    )
                   )
                 }
               })}
             </Box>
+            <Box height={isArtistsDropdown ? "90px" : "auto"}>
+              <ViewAllMenuItem key={lastMenuItem.text} href={lastMenuItem.href}>
+                {lastMenuItem.text}
+              </ViewAllMenuItem>
+            </Box>
           </Links>
 
-          {menu.links.map(subMenu => {
-            if ("menu" in subMenu) {
-              return <DropDownSection key={subMenu.text} section={subMenu} />
-            }
-          })}
+          <Flex flexDirection="column" pb={3}>
+            <Flex>
+              {menu.links.map(subMenu => {
+                if ("menu" in subMenu) {
+                  return (
+                    <DropDownSection key={subMenu.text} section={subMenu} />
+                  )
+                }
+              })}
+            </Flex>
+            {isArtistsDropdown && (
```

Comment:
@anastasiapyzhik - this `isArtistsDropdown` will always be true, as what is assigning it up above is a constant. Is this intended?

Reply:
It won't always be true as there might be another `contextModule` value, just like here https://github.com/artsy/force/blob/8d9ab79abb870b4fd6d171a7ed7fdbdfcad5a8e2/src/v2/Components/NavBar/NavBar.tsx#L225-L227

Reply:
Ah, i see 👍, cool